Vagrantfile

```diff
@@ -20,11 +20,14 @@ Vagrant.configure(2) do |config|
     config.vm.provision "shell", inline: <<-EOF
         set -eu
 
-        sudo dnf install -y cockpit
+        sudo dnf install -y cockpit podman
 
         printf "[WebService]\nAllowUnencrypted=true\n" > /etc/cockpit/cockpit.conf
 
         systemctl enable cockpit.socket
         systemctl start cockpit.socket
+
+        systemctl enable io.podman.{socket,service}
+        systemctl start io.podman.{socket,service}
```

Comment:
Please only enable the .socket. The .service really shouldn't have an `[Install]` section in the first place (this is a podman bug).